cfe_internal/enterprise/federation/federation.cf

```diff
@@ -342,10 +345,8 @@ bundle agent clean_when_off
 
   commands:
       # Oh, the humanity! Where for art thou databases: promises for psql!
-      `$(sys.bindir)/psql cfsettings --quiet --command "TRUNCATE TABLE remote_hubs"` -> { "ENT-7233" }
-        if => isgreaterthan( "$(remote_hubs_table_row_count)", "0" );
-      `$(sys.bindir)/psql cfsettings --quiet --command "TRUNCATE TABLE federated_reporting_settings"` -> { "ENT-7233" }
-        if => isgreaterthan( "$(federated_reporting_settings_table_row_count)", "0" );
+      `$(sys.bindir)/psql cfsettings --quiet --command "TRUNCATE TABLE remote_hubs"` -> { "ENT-7233" };
+      `$(sys.bindir)/psql cfsettings --quiet --command "TRUNCATE TABLE federated_reporting_settings"` -> { "ENT-7233" };
```

Comment:
I don't think this should be changed.

Reply:
these vars `federated_reporting_settings_table_row_count` and `remote_hubs_table_row_count` are not defined (that's why tables aren't truncated), and these if conds make no sense from my point of view.

Reply:
I found them above:

```
 "remote_hubs_row_count" string => execresult(`$(sys.bindir)/psql cfsettings --quiet --tuples-only --command "SELECT COUNT(*) FROM remote_hubs" 2>/dev/null`, useshell);
    "federated_reporting_settings_row_count" string => execresult(`$(sys.bindir)/psql cfsettings --quiet --tuples-only --command "SELECT COUNT(*) FROM federated_reporting_settings" 2>/dev/null`, useshell);
```

but names are still wrong.

I will rework it. it makes sense to make masterfiles backward compatible but the queries are wrong, select return error if a table is not exists

Reply:
removed vars and reverted IFs